### PR TITLE
README FAQ - first question

### PR DESCRIPTION
**Pull request by jacortinas**
Add a small FAQ section to the README.

### CODE REVIEW

**Comment by reneklacan** [comment]
👍

**Comment by jacortinas** [comment]
Change suggested in #7

File: README.md [hunk 1]
```diff
@@ -60,7 +60,11 @@ If you find a bug please start by reading through the our current list of [open
 
 Woot woot! Please checkout our [Contributing Guidelines](https://github.com/fetlife/fetlife-ios/blob/master/CONTRIBUTING.md) and go from there.
 
+### Frequently Asked Questions
+
+- **Is a Mac required to install the application?** Yes a Mac computer with at least OS X 10.11 or later is required to run the app and install it on a device.
```
Comment (philippt) [inline README.md]: add a comma behind "Yes"?

**Comment by philippt** [comment]
:+1:

**Comment by jacortinas** [comment]
@JohnBaku you ok with the wording here?

**Comment by JohnBaku** [comment]
👍

**Pull request merged**